packages/autorest.python/test/azure/version-tolerant/Expected/AcceptanceTests/CustomPollerPagerVersionTolerant/custompollerpagerversiontolerant/operations/_operations.py

```diff
@@ -975,8 +975,8 @@ def get_single_pages_with_body_params(self, parameters: Union[JSON, IO], **kwarg
         }
         error_map.update(kwargs.pop("error_map", {}) or {})
         content_type = content_type or "application/json"
-        _json = None
-        _content = None
+        _json: Any = None
```

Comment:
nit: can we remove the type hints? they're not adding any value and it's hard to tell what code has actually changed

Reply:
updated.